Examples/Blink/Core/Src/main.cpp

```diff
@@ -19,6 +19,7 @@
 /* USER CODE END Header */
 /* Includes ------------------------------------------------------------------*/
 #include "main.h"
+#include "tim.h"

```

Comment:
Let's add some testing to the main here that we can test with down the line, we can chat about this if you're unsure